.github/workflows/qase.yml

```diff
@@ -0,0 +1,29 @@
+name: Qase Test Upload
+on:
+  push:
+    branches:
+      - 'main'
+    paths:
+      - '**/schemas/**'
+  
+jobs:
+  run-script:
+    name: update-qase
+    runs-on: ubuntu-latest
+    timeout-minutes: 1
```

Comment:
this seems pretty short, will this work for larger contributions too? or would 2 minutes be better?

Reply:
good question. I don't know! Maybe I should set at 5 minutes. Shouldn't hurt anything 🤷